pkg/controller/appdefinition/testdata/computeclass/all-set/existing.yaml

```diff
@@ -5,10 +5,6 @@ metadata:
   name: sample-compute-class
 description: Simple description for a simple ComputeClass
 cpuScaler: 0.25
-memory:
-  min: 1Mi
-  max: 2Mi
-  default: 1Mi
```

Comment:
Why is this changing? Seems like we were trying to test something here.

Reply:
It shouldn't be, just a typo left over it would seem.

Reply:
Thought I caught all of this already, my bad.